### PR TITLE
refactor(better): add new saved fields

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
SEED is improving integration with BETTER

#### What's this PR do?
- stores additional fields into the PropertyState from the BETTER analysis:
  - store each measure recommendation
  - store GHG reduction (note it's in kgCO2e not MTCO2e b/c that's what they give us)
  - min model R**2 (indicator of model quality)
  - SEED analysis ID (so users can go to that analysis to view more stuff about it)

#### How should this be manually tested?
Upload the file below, map all required fields. Import the meter data too. Select one or more properties and run the better analysis ([make sure your org settings has the token](https://github.com/SEED-platform/seed/pull/2788)).

Edit your inventory list columns settings to view the BETTER columns, these columns should be properly populated after running the analysis.

[PM_BETTER_PORTFOLIO_INPUT.xlsx](https://github.com/SEED-platform/seed/files/6932774/PM_BETTER_PORTFOLIO_INPUT.xlsx)

#### What are the relevant tickets?
#2778 

#### Screenshots (if appropriate)
New columns
Screen Shot 2021-08-04 at 12 02 25 PM

Inventory list with new columns
Screen Shot 2021-08-04 at 12 03 46 PM

### CODE REVIEW

File: seed/analysis_pipelines/better/pipeline.py [hunk 1]
```diff
@@ -300,7 +301,32 @@ def _process_results(self, analysis_id, progress_data_key):
...
+        'Reduce Equipment Schedules',
+        'Add Wall/Ceiling Insulation',
+        'Increase Cooling System Efficiency',
+        'Increase Heating System Efficiency'
+    ]
+    ee_measure_column_data_paths = [
+        ExtraDataColumnPath(
+            f'better_recommendation_{ee_measure_name.lower().replace(" ", "_")}',
+            f'BETTER Recommendation: {ee_measure_name}',
+            f'assessment.ee_measures.{ee_measure_name}'
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:325]: looks good. we'll eventually want to map these to the Measures world in SEED/BSync, but good for now.

File: seed/analysis_pipelines/better/pipeline.py [hunk 2]
```diff
@@ -312,8 +338,26 @@ def _process_results(self, analysis_id, progress_data_key):
...
+        ExtraDataColumnPath(
+            # we will manually add this to the data later (it's not part of BETTER's results)
+            # Provides info so user knows which SEED analysis last updated these stored values
+            'better_seed_analysis_id',
+            'BETTER Analysis Id',
+            'better_seed_analysis_id'
+        ),
+        ExtraDataColumnPath(
+            'better_min_model_r_squared',
+            'BETTER Min Model R^2',
```
Comment (nllong) [inline seed/analysis_pipelines/better/pipeline.py:355]: schweeet!

**Review by nllong — APPROVED** [review]
🔥!

There is a merge conflict with the chrome version. Didn't know how to resolve easily, so I'll leave that to you 🥴

File: seed/analysis_pipelines/better/pipeline.py (hunk 1 above)
Comment (macintoshpie) [reply seed/analysis_pipelines/better/pipeline.py:325]: 👍 Created this ticket to track: https://github.com/SEED-platform/seed/issues/2804

**Pull request merged**